controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -489,6 +481,14 @@ func (r *StorageConsumerReconciler) reconcileConsumerConfigMap(
 				return err
 			}
 		}
+
```

Comment:
The code below still have the issue of not guaranteeing that if there is a consumer controller it is the primary consumer